frontend/TestRun/ResultsGraphs.svelte

```diff
@@ -12,17 +12,19 @@
     let selectedTableFilters = [];
     let selectedColumnFilters = [];
     let filteredGraphs = [];
+    let startDate = "";
+    let endDate = "";
     let width = 500;  // default width for each chart
     let height = 300;  // default height for each chart
     const dispatch = createEventDispatcher();
 
     const dispatch_run_click = (e) => {
-        dispatch("runClick", {runId: e.detail.runId});
+        dispatch("runClick", { runId: e.detail.runId });
     };
 
     const fetchTestResults = async function (testId) {
         try {
-            let res = await fetch(`/api/v1/test-results?testId=${testId}`);
+            let res = await fetch(`/api/v1/test-results?testId=${testId}&startDate=${startDate}&endDate=${endDate}`);
```

Comment:
`query-string` package can be used here, optional at this stage, but it does save you from potential certain footguns later.